entity-framework/core/modeling/relational/tables.md

```diff
@@ -81,3 +81,6 @@ You can also specify a schema that the table belongs to.
         modelBuilder.Entity<Blog>()
             .ToTable("blogs", schema: "blogging");
 ````
+
+## Using Statement
+using System.ComponentModel.DataAnnotations.Schema;
```

Comment:
@divega Is DataAnnotations.Schema needed for the attribute code only / what's needed for the fluent API code?